frontend/services/src/stripe/stripeService.ts

```diff
@@ -56,3 +57,22 @@ export async function getStripeCheckoutFromEventId(eventId: EventId, isPrivate:
       return "/error";
     });
 }
+
+export async function getStripeAccId(userId: UserId): Promise<string> {
+  try {
+    if (!userId) {
+      throw Error(`getStripeAccId(${userId}): userId not valid`);
+    }
+    const userData = await getPrivateUserById(userId);
+    if (!userData) {
+      throw Error(`getStripeAccId(${userId}): private user data missing on userId`);
+    }
+    if (!userData.stripeAccount) {
+      return "";
+    }
+    return userData.stripeAccount;
+  } catch (e) {
+    stripeServiceLogger.error(e as string);
```

Comment:
we probably wanted something more descriptive here, but its ok thanks for converting it to logger